README.md

```diff
@@ -184,3 +184,17 @@ $this->app['db']->connection('couchbase')
     ->where('id', 1)
     ->returning(['id', 'name'])->get();
 ```
+
+## hacking
+
+To run tests there are should be following buckets created on local Couchbase cluster:
+
+``` php
+$cluster = new CouchbaseCluster('couchbase://127.0.0.1');
+$clusterManager = $cluster->manager('Administrator', 'password');
+$clusterManager->createBucket('testing', ['bucketType' => 'couchbase', 'saslPassword' => '', 'flushEnabled' => true]);
+$clusterManager->createBucket('memcache-couch', ['bucketType' => 'memcached', 'saslPassword' => '', 'flushEnabled' => true]);
```

Comment:
@ytake I'm not sure how you are initializing `couchbase-memcached`. You are specifying port 11255. Do you have standalone memcached process listening it?